content/updates/2021-11/1-TA-importexport.md

```diff
@@ -0,0 +1,20 @@
+---
+pim_announcement_img: ./img/TA_export.png
+pim_announcement_alt_img: "Table Attribute"
+pim_announcement_audience:
+- EE
+- GE
+---
+
+# Importing and Exporting Table Attributes
+::: meta-data type="New" features="Governance" available="Late-November" in="EE" link-to-doc="../articles/manage-multidimensional-data-in-a-table.html#importexport-your-table-data.html"
+
+Several dedicated import and export jobs for table attributes are available to adapt to your most efficient workflows. This means more autonomy for you and table attributes that become more readable and easier to enrich. Your exported table attributes will be displayed exactly the way you see them in the product edit form.  
```

Comment:
I don't feel like this paragraph is really telling me what is being released it's very benefits heavy. And the benefits listed at good! But tell me more about the "several dedicated import and export jobs" and by telling me what they are I'll have a better sense of how they "adapt to my most efficient workflows"